.env.example

```diff
@@ -2,6 +2,8 @@
 EXTERNAL_URL=
 SAML_AUDIENCE=https://saml.boxyhq.com
 JACKSON_API_KEYS="secret"
+ADMIN_PORTAL_TENANT="_jackson_boxyhq"
```

Comment:
Might be better to rename these `ADMIN_PORTAL_SSO_TENANT`, `ADMIN_PORTAL_SSO_PRODUCT`?